src/app/components/logo/SmallLogo.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react'
+import { FC } from 'react'
+import logo from './images/logo.png'
+
+export const SmallLogo: FC<{ size?: number }> = ({ size = 25 }) => (
+  <img src={logo} alt={'logo'} height={`${size}px`} width={`${size}px`} />
```

Comment:
- HTML height and width attrs should be defined as integer without a unit.
- alt value does not make sense + it should be taken from i18next